client/transport/src/web.rs

```diff
@@ -52,6 +52,11 @@ impl TransportSenderT for Sender {
 		Ok(())
 	}
 
+	async fn send_ping(&mut self) -> Result<(), Self::Error> {
+		tracing::trace!("send ping - not implemented for wasm");
+		Ok(())
```

Comment:
It feels like if you call this from wasm, it should result in an error, since calling it from wasm is pointless?

Reply:
yeah return an error here is better :1st_place_medal: